docs/src/tty.rst

```diff
@@ -25,12 +25,21 @@ Data types
     ::
 
       typedef enum {
+          /* Initial/normal terminal mode */
           /* Initial/normal terminal mode */
           UV_TTY_MODE_NORMAL,
-          /* Raw input mode (On Windows, ENABLE_WINDOW_INPUT is also enabled) */
+          /*
+          * Raw input mode (On Windows, ENABLE_WINDOW_INPUT is also enabled).
+          * Currently an alias for UV_TTY_MODE_RAW_LEGACY but may become an
+          * alias for UV_TTY_MODE_RAW_VT in libuv v2.x.
+          */
           UV_TTY_MODE_RAW,
           /* Binary-safe I/O mode for IPC (Unix-only) */
-          UV_TTY_MODE_IO
+          UV_TTY_MODE_IO,
+          /* Raw input mode. On Windows ENABLE_VIRTUAL_TERMINAL_INPUT is also set. */
+          UV_TTY_MODE_RAW_VT,
+          /* Alias for UV_TTY_MODE_RAW in libuv 1.x. */
+          UV_TTY_MODE_RAW_LEGACY
```

Comment:
I have a passionate dislike of anything with "legacy" in the name. Of aliases too, for that matter. Let's just stick with UV_TTY_MODE_RAW and UV_TTY_MODE_RAW_VT.